Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1661,6 +1661,21 @@ def setup_(self, step, stepName, stepDict, k, properties):
     offset = 0.404,
 )
 
+upgradeWFs['PatatrackPixelOnlyAlpakaCUDAValidation'] = PatatrackWorkflow(
+    digi = {
+        '--customise': 'HLTriggerOffline/Common/customizeHLTforAlpakavsCUDA.customizeHLTforAlpakavsCUDA',
+    },
+    reco = {
+        '-s': 'RAW2DIGI:RawToDigi_pixelOnly,RECO:reconstruction_pixelTrackingOnly,VALIDATION:@pixelTrackingOnlyValidation,DQM:@pixelTrackingOnlyDQM',
+        '--procModifiers': 'alpakaCUDAValidation'
+    },
+    harvest = {
+        '-s': 'HARVESTING:@trackingOnlyValidation+@pixelTrackingOnlyDQM'
+    },
+    suffix = 'Patatrack_PixelOnlyAlpakaCUDAValidation',
+    offset = 0.405,
```

Comment:
Could you pick a different number ?
The idea was to use 401-404 for quadruplets and 405-408 for triplets, to match the CUDA workflows (501-504 and 505-508).
Maybe `.5041`, because it's a validation workflow like `.404` and `.504`, and should be removed when the other `.50x` CUDA workflows are removed.